client/src/main/proto/spine/base/failure.proto

```diff
@@ -49,4 +50,11 @@ message Failure {
 
     // Additional information on the failure.
     map<string, google.protobuf.Value> attributes = 4;
-}
\ No newline at end of file
+}
+
+// A failure indicating that a message is invalid according to Spine custom Protobuf options.
```

Comment:
We should not mention the implementation details in this part of the documentation. I would also say that “... a message did not pass validation” or something like that.